### PR TITLE
Obey grammar geas

### PR DESCRIPTION
**Pull request by TheShadowFerret**
#### Summary
None

#### Purpose of change

Typos are bad.

#### Describe the solution

Fix typos.

#### Describe alternatives you've considered

There are no alternatives, the geas commands correct spelling!

#### Testing

N/A, it's a typo?

### CODE REVIEW

**Comment by John-Candlebury** [comment]
The enchantment description is missing an 'r' in monster if you can get it here I'd appreciate it.

File: data/mods/Isolation-Protocol/Player/Perks/enchantments.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "type": "enchantment",
     "id": "iso_ench_geas_buff",
     "name": { "str": "Ancient vow" },
-    "description": "Lightless monste!  I give you oblivion!\n\n Deal 15% more damage and regen 100% more stamine per turn when fighting high danger enemies.",
+    "description": "Lightless monster!  I give you oblivion!\n\n Deal 15% more damage and regen 100% more stamina per turn when fighting high danger enemies.",
```
Comment (Maleclypse) [inline data/mods/Isolation-Protocol/Player/Perks/enchantments.json:42]:
```suggestion
    "description": "Lightless monster!  I give you oblivion!\n\n Deal 15% more damage and regen 100% more stamina per turn when fighting high danger enemies."
```
Rebasing tests

File: data/mods/Isolation-Protocol/Player/Perks/enchantments.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
     "type": "enchantment",
     "id": "iso_ench_geas_buff",
     "name": { "str": "Ancient vow" },
-    "description": "Lightless monste!  I give you oblivion!\n\n Deal 15% more damage and regen 100% more stamine per turn when fighting high danger enemies.",
+    "description": "Lightless monster!  I give you oblivion!\n\n Deal 15% more damage and regen 100% more stamina per turn when fighting high danger enemies."
```
Comment (Maleclypse) [inline data/mods/Isolation-Protocol/Player/Perks/enchantments.json:42]:
```suggestion
    "description": "Lightless monster!  I give you oblivion!\n\n Deal 15% more damage and regen 100% more stamina per turn when fighting high danger enemies.",
```

Comment (TheShadowFerret) [reply data/mods/Isolation-Protocol/Player/Perks/enchantments.json]: I'll pretend I didn't see this.

File: data/mods/Isolation-Protocol/unique_items.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "color": "light_gray",
     "name": { "str": "repeating crossbow" },
     "reload_noise_volume": 4,
-    "description": "A crossbow fit with an automatic spanning mechanism that allows for the rapid firing of up to 4 bolts.\n\nIn an age long past, these weapons where issued to expeditions sent to contain a cursing plague in the northen reaches.",
+    "description": "A crossbow fit with an automatic spanning mechanism that allows for the rapid firing of up to 4 bolts.\n\nIn an age long past, these weapons where issued to expeditions sent to contain a cursing plague in the northern reaches.",
```
Comment (Night-Pryanik) [inline data/mods/Isolation-Protocol/unique_items.json:10]:
```suggestion
    "description": "A crossbow fit with an automatic spanning mechanism that allows for the rapid firing of up to 4 bolts.\n\nIn an age long past, these weapons were issued to expeditions sent to contain a cursing plague in the northern reaches.",
```

**Pull request merged**